src/app/components/PriceChart.tsx

```diff
@@ -316,3 +354,126 @@ export function PriceChart() {
     </>
   );
 }
+
+export function PairInfoTab() {
+  const t = useTranslations();
+  const { pairsList } = useAppSelector((state) => state.rewardSlice);
+  const selectedPairAddress = useAppSelector(
+    (state) => state.pairSelector.address
+  );
+  const pairInfo = pairsList.find(
+    (pairInfo) => pairInfo.address === selectedPairAddress
+  );
+
+  let name, address, orderReceiptAddress, token1, token2;
+  if (pairInfo) {
+    ({ name, address, orderReceiptAddress, token1, token2 } = pairInfo);
+  } else {
+    return "Selected pair not found in pairsList";
+  }
```

Comment:
This is a more concise syntax:
```suggestion
  if (!pairInfo) {
    return "Selected pair not found in pairsList";
  }
  const { name, address, orderReceiptAddress, token1, token2 } = pairInfo;
```